### DISCUSSION TITLE
Multiple peaks on posteriors distributions

### DISCUSSION THREAD

**Discussion by queili** [google/lightweight_mmm#97] [Q&A] [answered]
Hello everyone, 
I'm looking for some help in interpreting the posteriors distribution graph. 

**If we have multiple “hills” on the map, how should we interpret the plot?** 

![multiples hills](https://user-images.githubusercontent.com/114412380/199311620-97ed78de-eaa6-4045-bc0a-b14b0570ef85.png)

I believe lightweight will take the top of them, or am I wrong? Also, what does that suggest about my channel or the model itself? 

Thank you!

**Comment by michevan** [comment, marked as answer]
Hi @kayleighdp6! 

In general multiple peaks like this (bimodality or multi-modality) means there are multiple possible solutions that are all consistent with the data. It can be a sign of multicollinearity between some of your features. However, the peaks in your distributions don't look that large to me compared to the overall shape / width of the posterior distributions. One (speculative) idea I have would be that you might have a few widely-spaced values either for the data in these features or for your target variable, which could maybe produce a posterior distribution that looks like this.

LMMM will choose the mean or the median of these distributions (depending on the context), rather than the peak value, and the confidence intervals will account for this bumpy shape!

**Reply by queili** [reply]
Thank you very much, @michevan! It's much clearer now =)
